test/Concurrency/Runtime/taskgroup_cancelAll_cancellationHandler.swift

```diff
@@ -0,0 +1,40 @@
+// RUN: %target-run-simple-swift( -Xfrontend -disable-availability-checking %import-libdispatch -parse-as-library) | %FileCheck %s --dump-input always
```

Comment:
```suggestion
// RUN: %target-run-simple-swift( -Xfrontend -disable-availability-checking %import-libdispatch -parse-as-library) | %FileCheck %s 
```

was probably copied from another test, but we don't need to dump the output here